### ISSUE TITLE
[Todoist] Add NLP for task creation

### ISSUE THREAD

**Issue by mjtys** [raycast/extensions#12769]
### Extension

https://www.raycast.com/doist/todoist

### Description

Hey team,

It would be great to get NLP for the Todoist Add Task command. 

The way that I see it would be very similar to how Max's custom Notion extension works here: https://youtu.be/m5MDv9qwhU8?si=y-KUnLm8x8Btk7vs&t=296

### Who will benefit from this feature?

Everybody 🎉

**Comment by thomaslombart** [comment]
Thanks for the feature request. Would you like to work on it?

**Comment by jfkisafk** [comment]
I was looking into this feature too and if no one has started implementing this, I can take a stab at it if time permits. 

cc: @thomaslombart

**Comment by thomaslombart** [comment]
Sure, go ahead!

**Comment by adityaastro** [comment]
Hi All,

Is this feature still being worked on?

**Comment by jfkisafk** [comment]
Yes I have started working on this and picking up the pace soon.

**Comment by jfkisafk** [comment]
@raycastbot assign me

**Comment by adityaastro** [comment]
Hey Stelo,

Thanks a lot! I really appreciate your work on this extension, and for
taking up this issue.

Thanks & Regards,
Aditya Pethkar

**Comment by welkinbai** [comment]
@jfkisafk Thank you, I really need this feature🙏, looking forward to your good news

**Comment by jfkisafk** [comment]
Due to other obligations for the next month and a half, I might not be able to take a look into this and focusing on smaller fixes/feature requests for extensions. I will take a look after my obligations, if this is still open.

**Comment by thomaslombart** [comment]
Hey there, we've released AI extensions over the past few weeks that should make it easier to add NLP for task creation if you have Raycast Pro. You can use something like: `@todoist Buy groceries today at 10 in home project`.

**Comment by methilal** [comment]
With raycast's AI extension, does that mean the todoist extension won't get the NLP feature ultimately then?

**Comment by adityaastro** [comment]
If nobody is working on this, I will pick it up and try to implement this feature

**Comment by adityaastro** [comment]
@raycastbot assign me

**Comment by adityaastro** [comment]
This feature has now been added! Please provide me your feedbacks. Thanks a lot!
